support-frontend/assets/helpers/abTests/abtest.js

```diff
@@ -171,12 +175,33 @@ function randomNumber(mvtId: number, independent: boolean, seed: number): number
   return Math.abs(rng.int32());
 }
 
-function assignUserToVariant(mvtId: number, test: Test): string {
+const trackOptimizeExperiment = (optimizeId: string, variants: Variant[], variantIndex: number) => {
+  gaEvent(
+    {
+      category: 'ab-test-tracking',
+      action: optimizeId,
+      label: variants[variantIndex].id,
+    },
+    { // these map to dataLayer variables in GTM
+      experimentId: optimizeId,
+      experimentVariant: variantIndex,
+    },
+  );
+};
+
+function assignUserToVariant(
+  mvtId: number,
+  test: Test,
+): number {
   const { independent, seed } = test;
 
-  const variantIndex = randomNumber(mvtId, independent, seed) % test.variants.length;
+  return randomNumber(mvtId, independent, seed) % test.variants.length;
+}
+
+function targetPageMatches(targetPage: ?string) {
+  if (!targetPage) { return true; }
 
-  return test.variants[variantIndex].id;
+  return new URL(window.location).pathname.match(targetPage) != null;
```

Comment:
out of interest, why not just `window.location.pathname`?

Reply:
Not sure actually! I've just changed it